dev/io.openliberty.microprofile.telemetry.2.0.internal_fat_tck/publish/servers/Telemetry20LogsConfigTCKServer/bootstrap.properties

```diff
@@ -0,0 +1,22 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+#com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.microprofile.rest*=all:org.apache.cxf.*=all:com.ibm.ws.jaxrs*=all
+com.ibm.ws.logging.max.files=1
+com.ibm.ws.logging.trace.specification=*=info:TELEMETRY=all
+otel.trace.http.disabled=true
+otel.sdk.disabled=false
+otel.metrics.exporter=none
+otel.traces.exporter=none
+otel.logs.exporter=logging
+log.file.path=${server.output.dir}/logs/messages.log
```

Comment:
Is this for the TCK to know where to find and read the logs? If so I think it should be something like otel.tck.log.file.path

Reply:
It shouldn't be an otel config property. It is a separate property for the TCK: 

https://github.com/eclipse/microprofile-telemetry/blob/main/tck/logs/src/main/java/org/eclipse/microprofile/telemetry/logs/tck/application/JulTest.java#L57

Reply:
I mean that I think the TCK itself should use a property name like `otel.tck.log.file.path` to make sure it doesn't clash with a different thing also using `log.file.path`.